DatadogInternal/Sources/BacktraceReporting/BacktraceReporter.swift

```diff
@@ -6,13 +6,41 @@
 
 import Foundation
 
+/// A type representing unique thread identifier for `BacktraceReport` generation.
+public struct ThreadID {
+    public let thread_t: thread_t
+}
+
```

Comment:
Makes sense 👍. I ended up with using basic `typealias` for minimal clarity instead of adding new type:
```swift
/// A value identifying the thread for `BacktraceReport` generation.
public typealias ThreadID = thread_t
```